### ISSUE TITLE
Incorrectly hidden elements on page

### ISSUE THREAD

**Issue by ghost** [brave/adblock-lists#431]
`https://www.soulreaperzone.com/anime/re-zero-starting-life-in-another-world-season-2/`

When the adblocker is enabled the website hides the download buttons. Please fix it

**Comment by pes10k** [comment]
@Bluedot221 thank you for the bug report!

@antonok-edm the download buttons show in ublock origin.  Once you switch things over to those lists this week, can you either i) test and see if the download buttons appear in Brave, or ii) ping me and I can test the same?

**Comment by Bluedot221** [comment]
I reported it to the uBO maintainers and also to Adguard list maintainers and also to Nano adblocker developers. They fixed it. The download buttons appear when using uBO but not on Brave browser. I'm on Android with Brave browser version Version 1.10.99. I cannot install uBO on Brave browser on Android yet.

___

I have a lot of websites that are using anti-adblock. Should I list them here? Do I need to create seperate issues for each website or should I dump them all at once in a single issue? 

Thank a lot for helping.

**Comment by pes10k** [comment]
> I have a lot of websites that are using anti-adblock. Should I list them here? Do I need to create separate issues for each website or should I dump them all at once in a single issue?

If you have a list of these, of sites showing anti-adblock messages that Brave doesn't hide, then that would be really great to know, thanks!  No need to go to the trouble of creating a new issue for each of them, that sounds tedious.  But if you could create just one issue and list them there, we can take it from there (whether that means breaking the issues out on our side, or some other place for storing them).

@Bluedot221 thank you again for this help!

**Comment by ryanbr** [comment]
I just tested this site (Shields, aggressive), I get 2 download button links: @Bluedot221 

![download](https://user-images.githubusercontent.com/1659004/88910215-40b67d80-d2b0-11ea-9660-8a71f924d74e.png)

**Comment by ryanbr** [comment]
related report: https://github.com/NanoMeow/QuickReports/issues/2642 (From Jan 6th/2020)

**Comment by Bluedot221** [comment]
If it's not much to ask. Can you please try to download and follow the sites this website redirects to. It seems they are using link shorteners and redirects through a lot of websites before reaching the final destination. And in between you'll find a lot of unblocked ads and Anti adblockes too.

Further more if you try to download again from the first step it'll take to other link shorteners each time.


___
How can this force the the 'Agressive' mode on Android?

**Comment by ryanbr** [comment]
@Bluedot221   Android-side of things regarding cosmetic/snippets is still work in progress, just no ETA just yet. We're currently in progress developing more filter options in future Brave releases, making more insync with the default uBO filters.

 I've look at few sample links from soulreaperzone  (tested on Brave Desktop Release), both links seem to work well.

`https://sznlink.xyz/honzuki-no-gekokujou-2nd-season-12-end-erai-raws-multisubs-9012/
`https://tmearn.com/TEYeMO2`

**Comment by Bluedot221** [comment]
Okay, thanks for looking into it, can you alsohave a look at these, please:

`https://sznlink.xyz/hanamonogatari-bluray-mega-99616/` -- Anti adblock 

`https://tii.ai/J1ITSV4u` -- pop ups new tabs while solving captchas and after clicking 'Get link' button.

https://ouo.press/Fxsn1v -- click on 'I'm a human' and see an unblocked ad below the 'Get link'

**Comment by ryanbr** [comment]
Tested in Brave Desktop Release, didn't see any Anti-adblock warnings. I would test with uBO disabled, and default shields enabled. I did notice having uBo enabled with shields enabled caused some warnings to show. 

I have implement a further patch to fix some [revolving adservers](https://github.com/easylist/easylist/commit/13a469e3d0ac49a7bfc3871802fa923bb4937dad) which will help all platforms (ios/desktop/android).

**Comment by Bluedot221** [comment]
Will these patches update automatically or will they come with Brave browser updates?

**Comment by ryanbr** [comment]
These patches will be seen in the next 24-48hrs.   and for `ouo.press` https://github.com/easylist/easylist/commit/790c18b

Won't completely eliminate the popups, but it should help a little.

**Comment by ryanbr** [comment]
I'll check after the weekend, if everything looks good on this website I'll update/close this ticket.  Thanks for help and feedback @Bluedot221

**Comment by Bluedot221** [comment]
@ryanbr sorry for pinging you. 

I'm seeing ads on `https://animekaizoku.com/`. I don't see them on desktop but on Android they do. Can _nothing_ be done about them? It's the only reason I'm forced to use other browsers and switching default browser everytime is tedious.

![Screenshot_20200731_154327](https://user-images.githubusercontent.com/68992260/89025704-f7643d80-d344-11ea-8193-ec14af1e558e.jpg)

**Comment by ryanbr** [comment]
They're using revolving adservers, various random domains. Have added servers from them today/yesterday. Most of these changes will get to Ios and android. Ios adblock updates have been temporarily delayed (give it a few days or so).

Brave IOS may not be ideal for this site, given how many domain changes happening regularly to counter Easylist. Brave desktop has better options (snippets) to counter this.

**Comment by Bluedot221** [comment]
I was not taking about the `https://www.soulreaperzone.com` this site has its own problems, I don't expect much from this site. 

This is another site that I'm taking about: `https://animekaizoku.com/`

**Comment by ryanbr** [comment]
Okay, will look into it.

**Comment by Bluedot221** [comment]
Sorry, brave browser's Shields were off at that time when I visited `https://animekaizoku.com/`. Ads don't appear when Shields are on on this site. sorry again for interrupting you.

**Comment by ryanbr** [comment]
Closing ticket, both sites are working well. tag me if there is more issues @ghost

**Comment by Bluedot449** [comment]
@ryanbr anti-adblock here: `https://sznlink.xyz/re-zero-kara-hajimeru-isekai-seikatsu-2nd-season-06-multisubs-hevc-7464`.

Other than this everything seems to work fine.

Brave browser on Android: Version 1.12.113

![Screenshot_20200814_113413](https://user-images.githubusercontent.com/69662619/90218785-4a002800-de22-11ea-96a5-713cb7285658.jpg)

**Comment by ryanbr** [comment]
Android circumvention is limited, no easy fix for this site. Just disable site trackers for this site. just too many items to whitelist.

**Comment by FrostedIce339** [comment]
Okay, I just disabled `Block cross-site trackers` and the anti-adblock message is gone and there are no ads appearing as well. Thanks.

**Comment by Bluedot449** [comment]
Okay, I just disabled `Block cross-site trackers` and the anti-adblock message is gone and there are no ads appearing as well. Thanks.
